src/main/java/com/kodilla/ecommercee/domain/Delivery.java

```diff
@@ -0,0 +1,32 @@
+package com.kodilla.ecommercee.domain;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotNull;
+
+@AllArgsConstructor
+@NoArgsConstructor
+@Getter
+@Entity(name = "DELIVERIES")
```

Comment:
musimy też zdecydować się na tożsame nazewnictwo przy tabelach i kolumnach w projekcie (czy małe czy wielkie litery)